### PR TITLE
applications/misc: deprecate phases

### PR DESCRIPTION
**Pull request by Stunkymonkey**
###### Motivation for this change
#28910

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes (or backporting 21.05 Relase notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 132541` at ce7537a3 run on aarch64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>6 packages built successfully:</summary>
  <ul>
    <li>avrdudess</li>
    <li>emem</li>
    <li>ganttproject-bin</li>
    <li>gollum</li>
    <li>smos</li>
    <li>wavrsocvt</li>
  </ul>
</details>

File: pkgs/applications/misc/minergate/default.nix [hunk 1]
```diff
@@ -1,16 +1,16 @@
 { fetchurl, lib, stdenv, dpkg, makeWrapper, fontconfig, freetype, openssl, xorg, xkeyboard_config }:
 
 stdenv.mkDerivation {
-  version = "8.1";
+  version = "8.3";
   pname = "minergate";
   src = fetchurl {
     url = "https://minergate.com/download/ubuntu";
```
Comment (SuperSandro2000) [inline pkgs/applications/misc/minergate/default.nix:7]: Can you upload it to web.archive.org?

Comment (Stunkymonkey) [reply pkgs/applications/misc/minergate/default.nix:7]: does think [link](https://minergate.com/download/ubuntu) not work for you?

Comment (roberth) [reply pkgs/applications/misc/minergate/default.nix:7]: This url describes a 'latest version', which means that this package will break as soon as minergate updates. We need a url with immutable contents.

Comment (Stunkymonkey) [reply pkgs/applications/misc/minergate/default.nix]: is this fine?

Comment (Stunkymonkey) [reply pkgs/applications/misc/minergate/default.nix]: when visiting the site itself i do not see the link provided here. maybe this specific program `minergate` is no longer developed.
should we keep it? there is also `minergate-cli` with an equal comit history: initial commit from 2018 and the rest are treewide changes.

Comment (Stunkymonkey) [reply pkgs/applications/misc/minergate/default.nix]: lets merge it first and then discuss what to do about `minergate`

Comment (Stunkymonkey) [reply pkgs/applications/misc/minergate/default.nix]: what are we going to do about the `minergate` package?

Comment (SuperSandro2000) [reply pkgs/applications/misc/minergate/default.nix]: The URL should be changed to web.archive.org.

Comment (Stunkymonkey) [reply pkgs/applications/misc/minergate/default.nix]: the problem is: `archive.org` checks every file for viruses when uploading. some anti-virus scans detect mining software as viruses. archive.org then blocks the access to these files. so uploading them to archive.org is not really an option.

Comment (roberth) [reply pkgs/applications/misc/minergate/default.nix]: it leaves an impression of being abandoned. I've created https://github.com/NixOS/nixpkgs/issues/133127

Comment (SuperSandro2000) [reply pkgs/applications/misc/minergate/default.nix]: I am talking about web.archive.org, not archive.org. They are different services.

**Pull request merged**